lib/discourse_api/api/topics.rb

```diff
@@ -74,13 +74,13 @@ def delete_topic(id)
         delete("/t/#{id}.json")
       end
 
-      def topic_posts(topic_id, post_ids = [])
+      def topic_posts(topic_id, post_ids = [], params = {})
         url = ["/t/#{topic_id}/posts.json"]
```

Comment:
This is a great addition, thanks!

In order to document which params can be included can you add

```
params = API.params(params)
  .optional(:include_raw)
```

Check out the `update_topic_status`method  for an example

Reply:
Thanks for the reference @oblakeerickson. I think I documented everything I see in `Discourse` `topics#posts`